_artifacts/document-attributes-jbpm.adoc

```diff
@@ -24,6 +25,11 @@
 :CENTRAL: jBPM Workbench
 :URL_COMPONENT_CENTRAL: kie-wb
 :URL_COMPONENT_CENTRAL_UNDER: kie_wb
+:CENTRAL_CAPITAL_UNDER: KIE_WB
+:CENTRAL_ONEWORD: kiewb
+
 
 
 :ENGINE: jBPM engine
+:ASSET: process
```

Comment:
Your :ASSETS: attribute is problematic. I'd nix it and just say "services" throughout. Here's why.

The team convention we established a few months back with various SMEs (Edson and Matteo especially), Christy, and our team (you might recall the email thread), was that "services" was a good umbrella term that applies to all products and means roughly the following:

In DM/Drools, "services" = "decision services and other deployable assets"
In PAM/jBPM, "services" = "decision services, process applications, and other deployable assets"

This opposed to what we've been calling "assets" or "business assets", which are rules, process, cases, etc.

So in the packaging and deploying doc, for example, I handled it with one ifdef'ed portion in the doc to establish that definition and then used 'services" throughout:

For DM/Drools: "You deploy the packaged KJAR file to a Decision Server that runs the decision services and other deployable assets (collectively referred to as _services_)"

For PAM/jBPM: "You deploy the packaged KJAR file to a Process Server that runs the decision services, process applications, and other deployable assets (collectively referred to as _services_)"

I admit that the distinction still isn't rock solid, but either way, "decision services" isn't really interchangeable with "processes" as you have it. Rules, processes, cases, etc. are in one category, what we've been calling "business assets" or just "assets", and decision services, process applications, and planning solutions are another, what we've been calling "services". And we can't say "decisions services" in DM and then just "process applications" for PAM, because PAM also contains decision services. Hence the parent term "services".

This never made it to the [Glossary of terms](http://ccs-jenkins.gsslab.brq.redhat.com:8080/job/glossary-of-terms-and-conventions-for-product-documentation-branch-master/lastSuccessfulBuild/artifact/index.html#red_hat_jboss_brms_and_red_hat_jboss_bpm_suite), and only Emily seemed to show interest in the conversation at the time, so I'm not surprised if more writers are also out of the loop. People tend to not care until it matters to them (like now).

Your counter arguments are certainly welcome, but I think this actually simplifies things for you considerably, imo.